crates/matrix-sdk-crypto/src/verification/mod.rs

```diff
@@ -75,14 +75,16 @@ pub(crate) struct VerificationStore {
 /// [spec]: https://spec.matrix.org/unstable/client-server-api/#sas-method-emoji
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd)]
 pub struct Emoji {
+    /// The emoji code, for example: 0
+    pub code: &'static i32,
```

Comment:
```suggestion
    pub code: u32,
```

First, you don't need a reference to the value.

Second, `u32` is better than `i32` because the value is unsigned.